src/EntityId/StdinEntityIdListBuilder.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Wikibase\Import\EntityId;
+
+use RuntimeException;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Lucas Werkmeister < lucas.werkmeister@wikimedia.de >
+ */
+class StdinEntityIdListBuilder implements EntityIdListBuilder {
+
+	/**
+	 * @param string $input (ignored)
+	 *
+	 * @throws RuntimeException
+	 * @return string[]
+	 */
+	public function getEntityIds( $input ) {
+		$entityIds = [];
+		while ( $line = fgets( STDIN ) ) {
```

Comment:
Hm, that would look nice, but I don’t think it’s currently possible here, all the surrounding code expects arrays.